.github/workflows/main.yml

```
@@ -11,7 +11,7 @@ jobs:
 
       matrix:
         python-version: [3.6, 3.7, 3.8, 3.9]
-        qt-lib: [pyqt5, pyside2]
+        qt-lib: [pyqt5, pyside2, pyside6]
         os: [ubuntu-latest, windows-latest, macos-latest]
```

```suggestion
        os: [ubuntu-20.04, windows-latest, macos-latest]
```

unless @nicoddemus sees a reason why we should keep using Ubuntu 18.04 for the other environments - but "ubuntu-latest" will soon be 20.04 anyways: https://github.com/actions/virtual-environments/issues/1816